README.md

```diff
@@ -60,6 +60,15 @@ need reboot. If there are no other tasks in the playbook that require reboot,
 you can set this value to `true` and this role will handle the reboot for you,
 when needed.
 
+* `crypto_policies_transactional_update_reboot_ok`
+
+This variable is used to handle reboots required by transactional updates.
+If a transactional update requires a reboot, the role will proceed with the
+reboot if crypto_policies_transactional_update_reboot_ok is set to true. If set
+to false, the role will notify the user that a reboot is required, allowing
```

Comment:
I would propose to highlight the keywords/values in the sentence to make it easier to orient in the text and to match the existing documentation:
```suggestion
reboot if `crypto_policies_transactional_update_reboot_ok` is set to `true`. If set
to `false`, the role will notify the user that a reboot is required, allowing
```